src/protocol/utils.js

```diff
@@ -144,6 +144,10 @@ ArrayMap.prototype = {
   },
 };
 
+function compareBigInt(a, b) {
```

Comment:
Can this be compareNumericStrings or something, with a comment describing what's happening?

Reply:
I like compareNumericStrings